server/service.go

```diff
@@ -431,7 +445,33 @@ func (m *BoostService) handleGetPayload(w http.ResponseWriter, req *http.Request
 	requestCtx, requestCtxCancel := context.WithCancel(context.Background())
 	defer requestCtxCancel()
 
-	for _, relay := range m.relays {
+	key := bidRespKey{
+		slot:      payload.Message.Slot,
+		blockHash: payload.Message.Body.ExecutionPayloadHeader.BlockHash.String(),
+	}
+
+	// It means that someone is trying to call getPayload without having called getHeader first.
+	fmt.Printf("KEY %+v\n", key)
```

Comment:
remove debug leftovers :)

Reply:
Thanks ! Removed in 4cad6ecf18a1454f0ce34f95a1a9bb642bf0fd11.